spec/models/address_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Address, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # To do
```

Comment:
Não é interessante deixar comentários de implementações "futuras" em códigos de PRs, se algo for uma implementação futura é mais interessante abrir uma issue, caso não seja, basta apagar o comentário ou o arquivo temporariamente, mas não é bacana deixar a ideia de "pendências" em uma entrega completa de funcionalidade.

Reply:
Certo! Tínhamos apagado a linha com o pending para os testes ficarem menos poluídos, mas como o rubocop acusou uma ofensa, a de que não deveríamos deixar o teste em branco, nós adicionamos o comentário.